CtbGui

```diff
@@ -1989,24 +1991,9 @@ class MainWindow(QtWidgets.QMainWindow):
             print(message)
             return
 
+        self.analog_frame = decoder.moench04(analog_array)                
+        self.plotAnalogImage.setImage(self.analog_frame)
 
-        self.analog_frame = np.zeros((self.nAnalogCols, self.nAnalogRows))
-        adcNumbers = Defines.Moench04.adcNumbers
-        nPixelsPerSC = Defines.Moench04.nPixelsPerSuperColumn
-        scWidth = Defines.Moench04.superColumnWidth
-
-        for iPixel in range(nPixelsPerSC):
-            for iSC, iAdc in enumerate(adcNumbers):
-                col = ((iAdc % 16) * scWidth) + (iPixel % scWidth)
-                if iSC < 16:
-                    row = 199 - int(iPixel / scWidth)
-                else:
-                    row = 200 + int(iPixel / scWidth)
-                index_min = iPixel * 32 + iSC
-                pixel_value = analog_array[index_min]
-                self.analog_frame[row, col] = pixel_value
-                
-        self.plotAnalogImage.setImage(np.rot90(self.analog_frame, 3))
```

Comment:
this was removed because of merge conflict.  this had been updated in main and was not merged to your branch. So whats missing is the rotate 3 times to get 270 degrees. Will mention that in further whererever you do rotate 90.

Reply:
The question is if it's a matter of display, depending on where pyqtgraph puts the origin, or if it's the way they want the files. Anna's original decoder didn't do the rotate but maybe they always rotate when plotting?